src/tools/kg_tools.py

```
@@ -142,14 +144,144 @@ async def crawl_repo(ctx: Context, repo_url: str) -> str:
+                        # Create section nodes for documentation structure
+                        for i, section in enumerate(sections[:5]):  # Limit sections to avoid too many nodes
+                            section_node = StructuralNode(
+                                node_type="RepositorySection",
+                                node_id=f"{repo_url}#{chunk.source_path}#section_{i}",
+                                properties={
+                                    "repository": repo_name,
+                                    "title": section["title"],
+                                    "level": section["level"],
+                                    "content_preview": section["content"][:200],
+                                    "word_count": len(section["content"].split()),
+                                    "parent_file": chunk.source_path
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)